lib/src/Base/Func/ExpertMixture.cxx

```diff
@@ -34,22 +34,34 @@ ExpertMixture::ExpertMixture()
   : EvaluationImplementation()
   , experts_()
   , classifier_()
+  , supervised_(false)
```

Comment:
In the other constructor, supervised_ is true by default, it looks awkward to have false here.